### ISSUE TITLE
Fix CI upgrading to TF 2.4.1

### ISSUE THREAD

**Issue by MichaelBroughton** [tensorflow/quantum#465]
It looks like in #459 we accidentally introduced an a step in the CI that upgrades us to TF 2.4.1 instead of sticking with what we had installed in the requirements.txt file. We might want to make sure the versions stay aligned (it doesn't appear to be breaking anything now, but does make me a little uneasy).
![image](https://user-images.githubusercontent.com/2791105/105553819-fc35f880-5cba-11eb-9115-236001ceb5d7.png)

**Comment by jaeyoo** [comment]
```
When we release, lets also remember to update the examples in the research branch to use 2.4.1. Too (a seperate PR, but not urgent at all).
```

**Comment by MichaelBroughton** [comment]
This has been resolved now that we are depending on 2.4.1
